sdkconfig

```diff
@@ -53,6 +53,7 @@ CONFIG_BOOTLOADER_LOG_LEVEL=3
 CONFIG_BOOTLOADER_VDDSDIO_BOOST_1_9V=y
```

Comment:
Not quite sure why this got changed, but can you revert it?

Reply:
Resolved just the changes that were made to sdkconfig from that commit, everything still runs fine on my emulator-only end, i'm sure we will want to double check things.

Reply:
Ayep, definitely double checking over here